templates/stripes/stripe-table.php

```diff
@@ -0,0 +1,55 @@
+<?php
+$caption = !empty($caption["html"]) ? $this->inline($caption["html"]) : null;
+$headersInFirstRow = !empty($headersInFirstRow) ? $headersInFirstRow : false;
+$headersInFirstColumn = !empty($headersInFirstColumn) ? $headersInFirstColumn : false;
+
+// Ensure each item in the dataset is an array.
+$dataset = !empty($dataset) && is_array($dataset) ? $dataset : [[]];
+foreach ($dataset as $key => $value) {
+    if (!is_array($value)) {
+        $dataset[$key] = [];
+    }
+}
+?>
+
+<div class="<?= $this->e($type, 'wrapperClasses') ?>">
+  <div class="vssl-stripe-column">
+    <table>
+      <?php if (!empty($caption)) : ?>
+        <caption><?= $caption ?></caption>
+      <?php endif; ?>
```

Comment:
I'd expect the default caption behavior to be after the content—why do you like it here?

Reply:
I saw in Usage Notes here that it should be the first element in the table: 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption

But it can be positioned on the bottom with a specific css style property `caption-side: bottom;`